### PR TITLE
 chore: boxo: migrate from go-libipfs to boxo

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -140,9 +141,9 @@ require (
 	github.com/prometheus/client_golang v1.14.0
 	github.com/raulk/clock v1.1.0
 	github.com/raulk/go-watchdog v1.3.0
-	github.com/stretchr/testify v1.8.1
+	github.com/stretchr/testify v1.8.2
 	github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
-	github.com/urfave/cli/v2 v2.16.3
+	github.com/urfave/cli/v2 v2.25.0
```
[the remark this answers is not in the document]
Comment (guseggert) [reply go.mod:146]: (because it adds a --help flag on every command that didn't exist previously)

**Comment by BigLep** [comment]
Per 2023-04-11 Kubo maintainer conversation, @Jorropo is going to followup to make sure this get reviewed.

**Review by magik6k — COMMENTED** [review]
Looks good, one small issue

File: go.mod [hunk 2]
```diff
@@ -97,28 +99,27 @@ require (
...
 	github.com/ipfs/go-metrics-prometheus v0.0.2
-	github.com/ipfs/go-unixfs v0.4.3
-	github.com/ipfs/go-unixfsnode v1.5.2
+	github.com/ipfs/go-unixfs v0.4.5
+	github.com/ipfs/go-unixfsnode v1.6.0
 	github.com/ipfs/interface-go-ipfs-core v0.11.1
 	github.com/ipld/go-car v0.5.0
-	github.com/ipld/go-car/v2 v2.7.0
-	github.com/ipld/go-codec-dagpb v1.5.0
+	github.com/ipld/go-car/v2 v2.9.1-0.20230325062757-fff0e4397a3d
```
[the remark this answers is not in the document]
Comment (Jorropo) [reply go.mod]: Fixed ty

**Pull request merged**